queries/nu/highlights.scm

```diff
@@ -45,8 +45,8 @@
   "*"
   "="
   "!="
-  "&&"
-  "||"
+  "and"
+  "or"
```

Comment:
Did you also change the `and` and `or` operators in the `OPERATOR_PREC` table in `https://github.com/petrisch/tree-sitter-nu/grammar.json`? If not, this would be a good start. You'll likely also need to recompile the grammar.

Reply:
I did now and I also forgott to change it in the grammar.js as well.
Also I changed the && and || appearences in node-types.json, but I don't really know what those are doing.

I still have a _ts_parse_query error thou.
When searching for more && appearences I find it still in parser.c, but if I am correct, this file gets compiled away when I recompile it in nvim.

There is still something missing and I have to spend more time on it.
I have a highlighting working on Windows, but the same fails in Linux, which is odd.
But that must be something from my setup/versioning things.

Reply:
Changes in the grammar.js are enough. You can generate the parser and node-type files via `tree-sitter generate`.

I've made the changes as you suggested and updated the tree-sitter repo. Parsing of `and` and `or` should work now (after reinstalling the nu tree-sitter of course).

Thanks for your pull request and the suggested change.